src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AttendanceController.java

```diff
@@ -40,7 +43,19 @@ public void postAttendance(
             @RequestBody
             @Valid final CreateAttendanceDto attendance) {
 
-        this.attendanceService.createOffenderAttendance(attendance);
+        this.attendanceService.createAttendance(attendance);
+    }
+
+    @PutMapping(path = "/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
don't think this `@ResponseStatus` line is doing anything since you're returning a `ResponseEntity` anyway?